src/main/java/de/bsi/secvisogram/csaf_cms_backend/service/AdvisoryService.java

```diff
@@ -78,35 +58,61 @@ public Long getAdvisoryCount() {
      *
      * @return a list of information objects
      */
-    public List<AdvisoryInformationResponse> getAdvisoryIds() {
-
-        List<String> infoFields = List.of(
-                WORKFLOW_STATE_FIELD,
-                OWNER_FIELD,
-                TYPE_FIELD,
-                String.join(".", DOCUMENT_TITLE),
-                String.join(".", DOCUMENT_TRACKING_ID),
-                CouchDbService.REVISION_FIELD,
-                CouchDbService.ID_FIELD
+    public List<AdvisoryInformationResponse> getAdvisoryInfromations() {
```

Comment:
```suggestion
    public List<AdvisoryInformationResponse> getAdvisoryInformations() {
```

This suggestion alone won't do, it needs to be renamed at all places it is used.

Reply:
fixed in 993332eaefd0e18f45ac4399c793ef38e4444c8a